README_ESPRESSO.md

```diff
@@ -15,95 +15,8 @@
 
 * Enter the nix shell of this project
 
-> nix develop .
+> nix develop
```

Comment:
I might be wrong, but I thought with `.`, it would explicitly use `flake.nix` in the current directory, whereas without `.`, the nearest `flake.nix` would be used, which might not be the correct one in some cases. The former seems like a better practice.